vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/main/java/com/vaadin/flow/component/combobox/test/template/ValidationConnectorErrorPage.java

```diff
@@ -1,33 +0,0 @@
-package com.vaadin.flow.component.combobox.test.template;
-
-import com.vaadin.flow.component.HasComponents;
-import com.vaadin.flow.component.Tag;
-import com.vaadin.flow.component.combobox.ComboBox;
-import com.vaadin.flow.component.combobox.bean.SimpleBean;
-import com.vaadin.flow.component.dependency.JsModule;
-import com.vaadin.flow.component.html.Div;
-import com.vaadin.flow.component.polymertemplate.Id;
-import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
-import com.vaadin.flow.data.binder.Binder;
-import com.vaadin.flow.router.Route;
-import com.vaadin.flow.templatemodel.TemplateModel;
-
-@Tag("validation-connector")
-@JsModule("./src/validation-connector.js")
-@Route("vaadin-combo-box/validation-connector")
-public class ValidationConnectorErrorPage extends PolymerTemplate<TemplateModel>
```

Comment:
This was a regression test for a client-side script error that occured when enabling required validation through the connector. Obsolete as the connector calls have been removed.